src/main/steps/privacy-policy/content.ts

```diff
@@ -1,207 +1,270 @@
-  writeTo: 'Ysgrifennwch i: Post Point 10.38, 102 Petty France, Llundain SW1H 9AJ',
-  email:
-    'anfonwch e-bost i: <a href="mailto:data.compliance@justice.gov.uk" class="govuk-link" aria-label="Bydd y ddolen hon yn agor neges e-bost newydd i HMCTSforms@justice.gov.uk">data.compliance@justice.gov.uk</a>',
-  youCanAlsoComplain:
-    "Gallwch yn ogystal gyflwyno cwyn i <a class='govuk-link' href='https://ico.org.uk/global/contact-us' target='_blank'>Swyddfa’r Comisiynydd Gwybodaeth</a> os ydych yn anfodlon â’n hymateb neu’n credu nad ydym yn prosesu eich data personol yn gyfreithlon.",
-  sharingYourData: 'Rhannu eich data',
-  whenYouUseDivorceService: 'Mae arnom angen y data personol a ganlyn ar gyfer y gwasanaeth hwn:',
-  applicantAndRespNames: 'Enwau cyfredol y ceisydd a’r atebydd',
-  changedName: 'Tystiolaeth o newid enw’r ceisydd os yw wedi newid ei enw ers priodi',
-  marriageCert: 'Copi o’r dystysgrif briodas a’r manylion sydd ynddi',
-  countryInMarried: 'Y wlad y mae’r ceisydd a’r atebydd yn preswylio ynddi’n arferol',
-  yourHabitualResidence: 'A yw’r ceisydd a’r atebydd yn preswylio’n arferol yng Nghymru neu Loegr ai peidio?',
-  yourDomicile: 'A yw’r ceisydd a’r atebydd â’u domisil yng Nghymru neu Loegr ai peidio?',
-  emailAndPhone: 'Cyfeiriadau, cyfeiriadau e-bost a rhifau ffôn y ceisydd a’r atebydd',
-  addressAndNames: 'Cyfeiriadau ac enwau cyfreithwyr y ceisydd a’r atebydd (os oes ganddynt gyfreithwyr)',
-  reasonsForDivorce: 'Manylion ynglŷn â’r rheswm dros ysgaru',
-  otherCourtCases: 'Manylion achosion llys eraill y bu a wnelo’r ceisydd a’r atebydd â hwy',
-  adulteryNames:
-    'Enw a chyfeiriad yr unigolyn yr honnir bod yr atebydd wedi godinebu ag ef/hi (dewisol ac ni ofynnir y cwestiwn ond pan ddywedir mai godineb yw’r rheswm dros ysgaru)',
-  receivingNotifications: 'Derbyn hysbysiadau',
-  youNeedSignUp:
-    'Mae angen ichi gofrestru i dderbyn hysbysiadau i ddefnyddio’r gwasanaeth ysgaru. Mae hwn yn ofyniad cyfreithiol er mwyn i’r cais am ysgariad fynd yn ei flaen.',
-  askedForEmailAddress:
-    'Pan fyddwch yn defnyddio’r gwasanaeth hwn gofynnir ichi ddefnyddio’ch cyfeiriad e-bost i greu cyfrif. Byddwch yn gallu defnyddio’r neges e-bost hwn a chyfrinair i fewngofnodi i wasanaethau eraill GLlTEM.',
-  holdYourData:
-    'Tra byddwch yn llenwi cais am ysgariad neu’n ymateb i gais, byddwn yn cadw eich data am hyd at 6 mis. Os na fyddwch yn cwblhau’r cais yn ystod yr amser hwn, bydd yn rhaid ichi ddechrau eto.',
-  whenADivorceIsFinalised:
-    'Pan gwblheir ysgariad cedwir yr achos am 18 mlynedd. Ar ôl y cyfnod hwnnw, dilëir peth data (o’r dyfarniad nisi a’r dyfarniad absoliwt).',
-  theRemainder:
-    'Cedwir gweddill gwybodaeth yr achos am 82 blynedd arall. Wedi cyfanswm o 100 mlynedd dileir y data hwnnw.',
-  thePersonal: 'Y data personol mae arnom ei angen',
-  storingYourData: 'Storio eich data',
-};
+export const generateContent = ({ isDivorce }: { isDivorce: boolean }): Translations => {
```

Comment:
```suggestion
export const generateContent: TranslationFn = ({ isDivorce }) => {
```

Reply:
Already added the import, just githhub didnt put it together nicely, [same pr](https://github.com/hmcts/nfdiv-frontend/pull/282/files/0d568be892904e375324f382229fdae88709be34#diff-12c5d153250d9cdd9409512f123d7a59c495c7291c1f43a4ef39be4740812007R1)